main.py

```diff
@@ -69,6 +70,11 @@
         im = cv.medianBlur(mat_img, 3)
 
         # Reading text, searching for keywords
+        
```

Comment:
Hey Jérôme, I'm thinking, we can even include the first pass in the loop and it would make the code more concise. Do you mind doing that modification? Thanks.